Dockerfile

```diff
@@ -127,3 +128,5 @@ ENV JAVA_SEMERU17_HOME=/usr/lib/jvm/semeru17
 
 ENV JAVA_GRAALVM11_HOME=/usr/lib/jvm/graalvm11
 ENV JAVA_GRAALVM17_HOME=/usr/lib/jvm/graalvm17
+
+ENV JAVA_UBUNTU17_HOME=/usr/lib/jvm/java-17-openjdk-amd64/bin/java
```

Comment:
This should be just `/usr/lib/jvm/java-17-openjdk-amd64`
or even better, `/usr/lib/jvm/ubuntu17` to align with the others.

Reply:
sorry this was copy pasta from where I set this up elsewhere